src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -9495,7 +9405,7 @@ static void Main()
         R a = new R();
         R b = new R();
         R c = new R();
-        R d;
+        scoped R d;
```

Comment:
>scoped [](http://example.com/codeflow?start=8&length=6)

We should also test the case where the `out` local is not marked `scoped`.

Reply:
Added `OutReturnOnly_5` which enumerates the different cases here.